src/test/java/com/esri/core/geometry/TestPolygon.java

```diff
@@ -1318,6 +1318,28 @@ public void testSegmentIteratorCrash() {
 		assertEquals(segments, 15);
 	}
 
+	@Test
+	public void testIsExteriorRing() {
+		{
+			Polygon poly = new Polygon();
+			poly.startPath(0, 0);
+			poly.lineTo(0, 1);
+			poly.lineTo(1, 1);
+			poly.lineTo(1, 0);
+			assertTrue(poly.isExteriorRing(0));
+		}
+
+		{
+			Polygon poly = new Polygon();
+			poly.startPath(-135, 85);
+			poly.lineTo(-45, 85);
+			poly.lineTo(45, 85);
+			poly.lineTo(135, 85);
+			assertTrue(poly.isExteriorRing(0));
```

Comment:
This polygon is not OGC valid. It has a zero area. If you make it OGC valid it would disapear.